ros2topic/ros2topic/verb/info.py

```diff
@@ -30,6 +31,21 @@ def add_arguments(self, parser, cli_name):
     def main(self, *, args):
         with DirectNode(args) as node:
             topic_name = args.topic_name
+            type_name = "NOT FOUND!"
```

Comment:
In the case of the topic not being found I would rather expect a non-zero return code (instead of the usual output with a different string as `Type: `).